[2025-01-31T05:24:25.107+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag1_v5.third_task scheduled__2024-05-02T00:00:00+00:00 [queued]>
[2025-01-31T05:24:25.154+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag1_v5.third_task scheduled__2024-05-02T00:00:00+00:00 [queued]>
[2025-01-31T05:24:25.154+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 6
[2025-01-31T05:24:25.230+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): third_task> on 2024-05-02 00:00:00+00:00
[2025-01-31T05:24:25.255+0000] {standard_task_runner.py:57} INFO - Started process 23308 to run task
[2025-01-31T05:24:25.263+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag1_v5', 'third_task', 'scheduled__2024-05-02T00:00:00+00:00', '--job-id', '7332', '--raw', '--subdir', 'DAGS_FOLDER/dag_example.py', '--cfg-path', '/tmp/tmpttf2opfh']
[2025-01-31T05:24:25.270+0000] {standard_task_runner.py:85} INFO - Job 7332: Subtask third_task
[2025-01-31T05:24:25.744+0000] {task_command.py:415} INFO - Running <TaskInstance: dag1_v5.third_task scheduled__2024-05-02T00:00:00+00:00 [running]> on host f8bdfcb8bca5
[2025-01-31T05:24:26.287+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='anhtupham99' AIRFLOW_CTX_DAG_ID='dag1_v5' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-02T00:00:00+00:00'
[2025-01-31T05:24:26.290+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-01-31T05:24:26.291+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo I am the third task, I run after the task 1, at the same time with task 2']
[2025-01-31T05:24:26.349+0000] {subprocess.py:86} INFO - Output:
[2025-01-31T05:24:26.353+0000] {subprocess.py:93} INFO - I am the third task, I run after the task 1, at the same time with task 2
[2025-01-31T05:24:26.359+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-01-31T05:24:26.432+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1_v5, task_id=third_task, execution_date=20240502T000000, start_date=20250131T052425, end_date=20250131T052426
[2025-01-31T05:24:26.505+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-01-31T05:24:26.591+0000] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
